uaaextras/clients/cf.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from cloudfoundry_client.client import CloudFoundryClient
+from cloudfoundry_client.errors import InvalidStatusCode
+
+logger = logging.getLogger(__name__)
+
+class CFClient(object):
+    """
+    A minimal client for CF
+    """
+
+    def __init__(self, target_endpoint, cf_user, cf_pass):
+        self.target_endpoint = target_endpoint
+        self.cf_user = cf_user
+        self.cf_pass = cf_pass
+
+    def _get_cf_client(self):
+        client = CloudFoundryClient(self.target_endpoint)
+        client.init_with_user_credentials(self.cf_user, self.cf_pass)
```

Comment:
is it possible we could use the token the user gave us here, so we don't have to involve another set of credentials?

Reply:
it looks like yes: https://github.com/antechrestos/OAuth2Client/blob/master/main/oauth2_client/credentials_manager.py#L146

Reply:
Updated to use access token